bin/cron/update_project_count

```diff
@@ -21,9 +21,9 @@ MAX_EXECUTION_TIME = 1_200_000
 MAX_EXECUTION_TIME_SEC = MAX_EXECUTION_TIME / 1000
 
 # Connection to read from Pegasus reporting database.
-PEGASUS_DB_READER = sequel_connect(
-  CDO.pegasus_db_reader,
-  CDO.pegasus_db_reader,
+DASHBOARD_DB_READER = sequel_connect(
+  CDO.dashboard_db_reader,
```

Comment:
Just double checking: is this the way to connect to the reporting database like the comment says?

Reply:
Looks like note but it also looks like the comment might be outdated?

Reply:
Yeah, I think the comment is out-of-date because `pegasus_db_reader` isn't the reporting DB. I'll update the comment